src/main/java/Jarvis/Ui.java

```diff
@@ -0,0 +1,129 @@
+package Jarvis;
+
+import java.util.ArrayList;
+
+public class Ui {
+    private static final String name = "Jarvis";
+    private static final String line = "____________________________________________________________";
+    private static final String greeting = "Good day Sir! I'm ";
+    private static final String question = "How can I help you today Sir?";
+    private static final String signOff = "Good bye Sir!";
+    private static final String listInforming = "Here are the tasks in your list Sir:";
+    private static final String markInforming = "Roger that Sir, I've marked this task as done:";
+    private static final String uncheckInforming = "Noted Sir, I've marked this task as NOT done yet:";
+    private static final String taskInforming = "As you please Sir, I've added the task:";
+    private static final String deleteInforming = "Alright Sir, I've removed this task";
+    private static final String findInforming = "Here are the matching tasks in your list Sir:";
```

Comment:
Given that these are meant to be constants, do consider naming them using the constant names style stated in the Java coding standard for this course!
```suggestion
    private static final String NAME = "Jarvis";
    private static final String LINE = "____________________________________________________________";
    private static final String GREETING_MESSAGE = "Good day Sir! I'm ";
    private static final String QUESTION_MESSAGE = "How can I help you today Sir?";
    private static final String SIGN_OFF_MESSAGE = "Good bye Sir!";
    private static final String LIST_MESSAGE = "Here are the tasks in your list Sir:";
    private static final String MARK_MESSAGE = "Roger that Sir, I've marked this task as done:";
    private static final String UNCHECK_MESSAGE = "Noted Sir, I've marked this task as NOT done yet:";
    private static final String TASK_MESSAGE = "As you please Sir, I've added the task:";
    private static final String DELETE_MESSAGE = "Alright Sir, I've removed this task";
    private static final String FIND_MESSAGE = "Here are the matching tasks in your list Sir:";
```